README.md

```diff
@@ -1,7 +1,8 @@
 # Collector for Confluence
 Collects meta-information about conducted threat modeling activities from Confluence wiki pages.
 
-## Meta-information about a Threat Modeling Activity
+## Meta-information about a DateTitleLink
+The generic type DateTitleLink can be used to assess DSOMM activities like Conduction ofThreat Modeling or Conduction of Pentests.
```

Comment:
Great to explain the scope and generic concept of *DSOMM activities* here :green_heart: 

### Naming
To make it generic, we can often just remove the specifier .. see what's left then, the generic `Activity` 

> ## Meta-information about a ~~Threat Modeling~~ Activity

What is composed of those 3 parts `Date`, `Title`, `Link` ?
> ## Meta-information about a ~~DateTitleLink~~ 
>
> The generic type `DateTitleLink` can be used to assess *DSOMM activities*, like Conduction ofThreat Modeling or Conduction of Pentests.

See highlighted above, an _event_ or an _activity_ within the context of our DSOMM .. so I would name the concept just `Activity`, example:
```suggestion
## Meta-information about a DSOMM Activities
The generic type `Activity` can be used to assess DSOMM activities like "Conduction of Threat Modeling" or "Conduction of Pentests".
```

Reply:
It doesn't fit for all DSOMM activities, but I am ok to name it activity here to make it easier to understand within this scope.